pkg/ingester/shipper.go

```diff
@@ -25,10 +25,16 @@ import (
 	"github.com/prometheus/prometheus/tsdb/fileutil"
 	"github.com/thanos-io/objstore"
 
+	mimir_tsdb "github.com/grafana/mimir/pkg/storage/tsdb"
 	"github.com/grafana/mimir/pkg/storage/tsdb/block"
 	"github.com/grafana/mimir/pkg/storage/tsdb/metadata"
 )
 
+const (
+	// OutOfOrderExternalLabelValue is the value to be used for the OutOfOrderExternalLabelKey label
+	OutOfOrderExternalLabelValue = "true"
```

Comment:
I would suggest moving this to tsdb package as well, so that Label name and value are together. Alternatively, we can unexport it here.